src/containers/ResultsPanel/selectors.ts

```diff
@@ -149,7 +167,6 @@ const getShouldRenderTimeIndicator = (plot: RawPlotParams): boolean => {
 export const getPlotDataConfiguredForPlotly = createSelector(
     [getPlotData],
     (plotData: RawPlotParams[]): PlotConfig[] => {
-        if (!plotData) return [];
```

Comment:
This is duplicate logic so I removed it. TypeScript wouldn't allow a `null` or `undefined` or other falsy value to be passed in as `plotData`. And if `plotData` were an empty array, the `map` function below would just return an empty array anyway.